exercises/I-array-properties/exercise.js

```diff
@@ -5,15 +5,21 @@
 
 */
 
-function isEmpty(arr) {
-  return; // complete this statement
+function isEmpty() {
+  if ((isEmpty = numbers)) {
+    return false;
+  } else if (isEmpty = names) {
+    return true;
+    // complete this statement
+  }
+  console.log(isEmpty);
 }
```

Comment:
An easy way to check if an array is empty is to check if the length of the array is zero like so:
```
function isEmpty(arr) {
  if (arr.length === 0) {
    return true;
  } else {
    return false;
  }
}
```

Reply:
Great tip actually... thank you plenty...x